### PR TITLE
Highlight headers in PR template

### PR DESCRIPTION
**Pull request by Felixoid**
### Changelog category (leave one):
- Build/Testing/Packaging Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Highlight headers in the PR template. Improve description checking logging.

### CODE REVIEW

**Review by alexey-milovidov — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Changelog category (leave one):
 - Not for changelog (changelog entry is not required)
 
 
-Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
+## Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
```
Comment (azat) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: Maybe it is better to use `###`? To me `##` looks too big, especially for this line.

- `##`

## Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

- `###`

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Comment (Felixoid) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: Yep, looks good to me

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Changelog category (leave one):
+## Changelog category (leave one):
```
Comment (Felixoid) [inline .github/PULL_REQUEST_TEMPLATE.md:1]:
```suggestion
### Changelog category (leave one):
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Felixoid) [reply .github/PULL_REQUEST_TEMPLATE.md:12]:
```suggestion
### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
```

**Pull request merged**